demo/src/Components/Pills/Pills.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import T from 'prop-types'
+import classNames from 'classnames'
+
+import List from './List'
+import Panels from './Panels'
+
+class Pills extends Component {
+  constructor() {
+    super()
+
+    this.state = {
+      activeIndex: 0,
```

Comment:
This should be possible, but we only really care about this in Nebula React.  Nebula Components just ships Sass components and Nebula React ships the React components themselves.  The bits of React code are just a dumping ground to test out the components as they are built.  Ideally they can be deleted when build properly in Nebula-React and demoed in http://rbrtsmith.com/nebula/
Hope this makes sense!